### PR TITLE
Stop ignoring 'build' dir when running Pytest

### PR DESCRIPTION
**Pull request by mikaell**
Update pytest.ini to remove 'build' from the ignored test directories.
Previous default setting was:

.*', 'build', 'dist', 'CVS', '_darcs', '{arch}', '*.egg', ’venv'

This PR fixes a bug.

It will start breaking some unit tests that previously were ignored.

This update is needed in every product using the tests/build name structure!

Further reading [https://docs.pytest.org/en/3.0.0/customize.html]

**How to test**:
1. how to install it
Update your repository

    $ git pull

1. how to test it, possibly with real cases/data
Run the pytest

    $ pytest tests

**Expected outcome**:
Pytest will now enter tests/build and execute unit tests. 

**Review:**
- [x] code approved by @moonso 
- [x] tests executed by @moonso 
- [ ] "merge and deploy" approved by

### CODE REVIEW

**Comment by dnil** [comment]
@moonso: please read through especially the gene / transcripts parts. There were a few things that looked kind of straight forward bugs once the build tests worked, but transcripts are inherently arcane arts so please read. In particular. there was an incomplete loop to set canonical_transcript, exon and some hgvs_name. Also what looked like a mixup of how inheritance models are encoded in variant-gene and panel-gene when testing building.

**Comment by moonso** [comment]
Sure I'll have a look

File: scout/parse/variant/gene.py [hunk 1]
```diff
@@ -96,6 +96,16 @@ def parse_genes(transcripts):
                     most_severe_polyphen = transcript['polyphen_prediction']
                     most_severe_region = SO_TERMS[consequence]['region']
 
+            if (transcript['is_canonical'] and transcript.get('coding_sequence_name')):
```
Comment (moonso) [inline scout/parse/variant/gene.py]: 👍

**Comment by moonso** [comment]
@northwestwitch you need to approve the changes that you requested before we can merge

**Comment by northwestwitch** [comment]
> @northwestwitch you need to approve the changes that you requested before we can merge

Done!

**Pull request merged**